app/services/alerts/adapters/analytics_adapter.rb

```diff
@@ -11,12 +11,13 @@ def report
       def produce_report(analysis_object)
         analysis_object.analyse(@analysis_date)
 
-        variables = if pull_variable_data?(analysis_object)
+        variables = if analysis_object.make_available_to_users?
```

Comment:
Is this reliable for all data, or just benchmark data?

If we run one alert - make_available_to_users? is the highest level flag to let us know everything is ok, great, but does this guarantee benchmark, priority, alert, chart data, rating, valid etc is all populated?